

import java.util.Scanner;

public abstract class Person {
    private String id;
    private String name;
    private int age;
    private String gender;

    public Person(String id) {
        this.id = id;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public abstract void NhapThongTin();

    public abstract void XuatThongTin();
}